src/chunk_adaptive.c

```diff
@@ -114,8 +114,8 @@ get_memory_cache_size(void)
  */
 #define DEFAULT_CACHE_MEMORY_SLACK 0.9
 
-static inline int64
-calculate_initial_chunk_target_size(void)
+extern inline int64
```

Comment:
```suggestion
inline int64
```

Reply:
It is already in the header: https://github.com/timescale/timescaledb/pull/1923/files#diff-4d56bc4bd354b0436991653168dabffeR30